experimental/Schemes/src/elliptic_surface.jl

```diff
@@ -2013,7 +2015,7 @@ function _pushforward_lattice_along_isomorphism(step::MorphismFromRationalFuncti
   for D in lat_X
     @assert length(components(D)) == 1 "divisors in the algebraic lattice must be prime"
     I = first(components(D))
-    @assert is_prime(I)
+    @assert I isa PrimeIdealSheafFromChart || has_attribute(I, :is_prime) "ideal sheaf must be known to be prime"
```

Comment:
This might be slightly dangerous. We cannot be sure that none ever sets a value of false to the attribute.... The value should still be checked for completeness.